docs/development/doc_guide.rst

```diff
@@ -1,76 +1,606 @@
-************************
-Documentation Guidelines
-************************
+*******************
+Documentation Guide
+*******************
 
-This document describes the documentation requirements and guidelines
-to be followed during the development of PlasmaPy and affiliated
-packages.
+Documentation that is up-to-date and understandable is vital to the
+health of a software project. This page describes the documentation
+requirements and guidelines to be followed during the development of
+PlasmaPy and affiliated packages.
+
+`PlasmaPy's online documentation`_ is hosted by `Read the Docs`_ and is
+available at these locations.
+
+* The documentation corresponding to the most recent official release
+  is labelled ``stable`` and is found at
+  `https://docs.plasmapy.org/en/stable/
+  <https://docs.plasmapy.org/en/stable/>`_.
+  The link `https://docs.plasmapy.org/ <https://docs.plasmapy.org/>`_
+  redirects to ``stable``.
+
+* The documentation corresponding to the ``main`` branch on
+  `PlasmaPy's GitHub repository`_ is labelled ``latest`` and can be
+  found at `https://docs.plasmapy.org/en/latest/
+  <https://docs.plasmapy.org/en/latest/>`_.
+
+A preview of the documentation is generated every time a pull request
+is created or updated. You can access this preview by scrolling down
+to the checks at the bottom of a pull request, and clicking on
+``Details`` next to ``docs/readthedocs.org:plasmapy``.
 
 Building documentation
 ======================
-Documentation is built from the main branch on every commit pushed
-to it.
 
-Sphinx, the documentation generator of PlasmaPy, uses reStructuredText (reST) as its markup language. A primer on reST is available at this `webpage
+To install all dependencies required to develop PlasmaPy on your local
+computer, enter the top-level directory of the cloned repository and
+run
+
+.. code-block:: bash
+
+   pip install tox -r requirements.txt
```

Comment:
Would it make sense to add `tox` to `requirements/extras.txt`?  We do end up using it for development quite a lot, and it's also not a large package.

Reply:
Why are you using `tox` here?  This can be done by just doing `pip install -r requirements.txt`. `tox` is not needed.

Reply:
With that said, yes `tox` should be added to the `extras.txt`.  Now I see why `tox` is in this code example.

I say add, `tox` to `extras.txt` and remove it from the code-block here.